dags/bqetl_backfill_complete.py

```diff
@@ -4,21 +4,23 @@
 
 from airflow import DAG
 from airflow.decorators import task, task_group
+from airflow.providers.slack.notifications.slack import send_slack_notification
 from airflow.providers.slack.operators.slack import SlackAPIPostOperator
 
 from operators.gcp_container_operator import GKEPodOperator
 from utils.tags import Tag
 
 AUTOMATION_SLACK_CHANNEL = "#dataops-alerts"
 SLACK_CONNECTION_ID = "overwatch_slack"
+DATA_PLATFORM_WG_CHANNEL_ID = "C01E8GDG80N"
 DOCKER_IMAGE = "gcr.io/moz-fx-data-airflow-prod-88e0/bigquery-etl:latest"
 
 tags = [Tag.ImpactTier.tier_3]
 
 default_args = {
     "email": [
         "ascholtz@mozilla.com",
-        "bewu@mozilla.com",
+        "benwu@mozilla.com",
```

Comment:
Changing my email alias because sendgrid seems to have bewu on some block list even after whd removed it